### ISSUE TITLE
Inconsistency in types of builds

### ISSUE THREAD

**Issue by brzys** [multitheftauto/mtasa-blue#1679]
**Describe the bug**
getVersion() returns incorrect build type for clients or builds are incorrectly classified.

**To reproduce**
1. Download newest nightly build https://nightly.mtasa.com/mtasa-1.5.7-rc-20651-20200923.exe
2. getVersion() returns 'Release' build type

**Expected behaviour**
https://wiki.multitheftauto.com/wiki/GetVersion highlights that `Nightly rX` is expected instead of `Release`
https://nightly.mtasa.com/ver/ shows that `Release` build is currently `1.5.7-9.20632.0`

**Screenshots**
![image](https://user-images.githubusercontent.com/5489020/94315730-6778f380-ff83-11ea-9a0c-0a92f78f90dc.png)

**Version**
Multi Theft Auto v1.5.7-release-20651

**Additional context**
Default build https://mtasa.com/ is 20651, which https://nightly.mtasa.com/ver/ clarifies as Nightly. What's going on?

**Comment by qaisjp** [comment]
When we say nightly builds these days, we are referring to the nightly 'stream' of builds. This is what nightly refers to here, on: https://nightly.mtasa.com/ver/. One way of thinking about it is that `Nightly` here https://nightly.mtasa.com/ver/ (and in the MTA settings) is "beta", and `Default` is "stable". 

Unfortunately it's been deleted, but https://nightly.mtasa.com/ used to describe the different kinds of builds we had. Nightly, unstable, experimental, release candidate (RC). That's what the 9 digit refers to.

We used to have Nightly builds which were bleeding edge *that couldn't connect to Release servers*. Then at some point, a while after the move to GitHub, we abandoned branches and decided to just make it possible for bleeding edge builds to connect to Release servers. To do that, we made our bleeding edge builds simply build as Release (well, RCs), instead of as Nightlies.


Long story short, MTA doesn't build any builds classified as "nightly" or "experimental" or "unstable" anymore, every build is a "release candidate" build. `getVersion()` refers to the kind of build (currently always RC), not the release stream ("stable" or "beta").

It's a bit silly.

There's some extra context here: https://github.com/multitheftauto/mtasa-blue/issues/1037
